docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -250,65 +227,419 @@ access_entries = {
 }
 ```
 
-In this updated configuration:
+In this configuration:
 
-- `principal_arn` should be replaced with the ARN of the IAM user or role.
-- `policy_associations` allow you to associate policies for fine-grained access control.
+- Replace `principal_arn` with the ARN of the IAM user or role.
+- Use `policy_associations` to define policies for fine-grained access control.
 
-For a list of policies, please visit the [AWS EKS Access Policies documentation](https://docs.aws.amazon.com/eks/latest/userguide/access-policies.html).
+For a full list of available policies, refer to the [AWS EKS Access Policies documentation](https://docs.aws.amazon.com/eks/latest/userguide/access-policies.html).
 
 :::info
+Please note that starting from version 2.x.x of this module, direct mappings through `aws_auth_roles` and `aws_auth_users` are no longer supported. If you are upgrading from version [1.x.x](https://github.com/camunda/camunda-tf-eks-module/releases/tag/1.0.3), you will need to fork the module and follow AWS's official instructions for managing the `aws-auth` ConfigMap.
 
-Please note that the version 2.x.x of this module no longer supports direct mappings via `aws_auth_roles` and `aws_auth_users`. If you are upgrading from version [1.x.x](https://github.com/camunda/camunda-tf-eks-module/releases/tag/1.0.3), fork the module repository and follow the official AWS instructions for managing the `aws-auth` ConfigMap.
 For more details, refer to the [official upgrade guide](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/UPGRADE-20.0.md).
+:::
+
+</p>
+</details>
+
+4. **Customize the cluster setup:**
+
+   The module offers various input options that allow you to further customize the cluster configuration. For a comprehensive list of available options and detailed usage instructions, please refer to the [EKS module documentation](https://github.com/camunda/camunda-tf-eks-module/blob/2.6.0/modules/eks-cluster/README.md).
+
+### PostgreSQL module setup
+
+:::info Optional module
+
+If you don't want to use this module, you can skip this section.
+However, please note that you may need to adjust the following instructions to remove references to this module.
+
+If you choose not to use this module, you'll need to either provide a managed PostgreSQL service or use the internal deployment by the Camunda Helm chart in Kubernetes.
 
 :::
 
-## Outputs
+The default PostgreSQL instance and database (`camunda`) created by this module is primarily intended for use with Keycloak. After the instance is set up, you may manually add additional databases for Identity multi-tenancy if needed, though this guide will not cover those steps, as the default configuration disables multi-tenancy.
```

Comment:
this needs rephrasing as it's now covered in `create the database and associated access`